test/e2e/localvolume_test.go

```diff
@@ -86,8 +86,8 @@ func LocalVolumeTest(ctx *framework.Context, cleanupFuncs *[]cleanupFn) func(*te
 			},
 		)
 
-		provisionerDSName := localVolume.Name + "-local-provisioner"
-		diskMakerDSName := localVolume.Name + "-local-diskmaker"
+		provisionerDSName := "local-provisioner"
```

Comment:
you can import these values from `nodedaemon` so we don't have to update in two places in the future (and after e2e fails once :) )